hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java

```diff
@@ -510,6 +512,28 @@ public static void addToOpenFileTable(OMMetadataManager omMetadataMgr,
             omFileInfo);
   }
 
+
+  /**
+   * Adding omKeyInfo to open file table.
+   *
+   * @param omMetadataMgr OM Metadata Manager
+   * @param batchOp       batch of db operations
+   * @param omFileInfo    omKeyInfo
+   * @param uploadID      uploadID
+   * @throws IOException DB failure
+   */
+  public static void addToOpenFileTable(OMMetadataManager omMetadataMgr,
```

Comment:
I didn't see this method is used in current PR change. Suppose this should be used in S3InitiateMultipartUploadResponseV1#addToDBBatch, right?

Reply:
Done!